src/core/scene/a-scene.js

```diff
@@ -147,6 +150,22 @@ var AScene = module.exports = registerElement('a-scene', {
       }
     },
 
+    promptForIOS: {
```

Comment:
I'm going to move this logic to its own third-party component because I want to add an "Add to Homescreen" toast message. and, it's way too hairy to add all that logic to A-Frame core IMO.